consensus-types/types/attestation_data.go

```diff
@@ -1,27 +1,22 @@
 // SPDX-License-Identifier: BUSL-1.1
 //
-// Copyright (c) 2025 Berachain Foundation
+// Copyright (C) 2025, Berachain Foundation. All rights reserved.
```

Comment:
https://github.com/berachain/beacon-kit/pull/2619 fixed the heading but not the rest of the licence